test/plugins/test/autolink.js

```diff
@@ -205,15 +210,33 @@ describe('AutoLink', function() {
 
 		testLink.call(this, {
 			expected:
-				'<p>invalid link <a href="http://ww.liferay.com">ww.liferay.com</a></p>',
-			html: '<p>invalid link ww.liferay.com { }</p>',
+				'<p>link <a href="http://ww.liferay.com">ww.liferay.com</a></p>',
+			html: '<p>link ww.liferay.com { }</p>',
 			keyCode: KEY_SPACE,
 		});
 
 		testLink.call(this, {
 			expected:
-				'<p>invalid link <a href="http://liferay.com">liferay.com</a></p>',
-			html: '<p>invalid link liferay.com { }</p>',
+				'<p>link <a href="http://liferay.com">liferay.com</a></p>',
+			html: '<p>link liferay.com { }</p>',
+			keyCode: KEY_SPACE,
+		});
+
+		testLink.call(this, {
+			expected: '<p>invalid link eeeewwww</p>',
+			html: '<p>invalid link eeeewwww { }</p>',
+			keyCode: KEY_SPACE,
+		});
+
+		testLink.call(this, {
+			expected: '<p>invalid link wwwwwwwww</p>',
+			html: '<p>invalid link wwwwwwwww { }</p>',
+			keyCode: KEY_SPACE,
+		});
+
+		testLink.call(this, {
+			expected: '<p>invalid link www www www www</p>',
+			html: '<p>invalid link www www www www { }</p>',
```

Comment:
Also would move inside "invalid" `it`.

Reply:
Yes, I'm gonna go ahead a put valid/invalid link tests in their respective "it" blocks